src/app/store/software-keys/software-key.actions.ts

```diff
@@ -43,12 +43,17 @@ function setWalletEncryptionPassword(args: {
       await checkForLegacyGaiaConfigWithKnownGeneratedAccountIndex(secretKey);
 
     async function doesStacksAddressHaveBalance(address: string) {
-      const resp = await stxClient.getAccountBalance(address);
+      const resp = await stxClient.getAccountBalance(address, new AbortSignal());
```

Comment:
I prob should have made signal arg as an optional, but when signal is optional - this leads to missing it in most queries...